src/test/java/com/virtuslab/archunit/UIThreadUnsafeMethodInvocationsTestSuite.java

```diff
@@ -97,6 +97,7 @@ public void check(JavaMethod method, ConditionEvents events) {
       "git4idea.GitUtil.getRepositories(com.intellij.openapi.project.Project)",
       "git4idea.GitUtil.getRepositoryManager(com.intellij.openapi.project.Project)",
       "git4idea.branch.GitBranchUtil.sortBranchNames(java.util.Collection)",
+      "git4idea.branch.GitBrancherImpl.createBranch(java.lang.String, java.util.Map<git4idea.repo.GitRepository,java.lang.String>, boolean)",
```

Comment:
Nope! you don't call this method anywhere directly, you call 

```suggestion
      "git4idea.branch.GitBrancher#createBranch(java.lang.String, java.util.Map<git4idea.repo.GitRepository,java.lang.String>)",
```

This is a whitelist of methods from git4idea that could be called even on UI thread

Reply:
Ok, I managed to add these method names properly

Reply:
In fact, it looks like we should use `.` instead of `#` in this place of code